### ISSUE TITLE
Next day forecast questions

### ISSUE THREAD

**Issue by skybullet1987** [awslabs/gluon-ts#988]
Good day! 

Thank you for a great forecasting tool, could you please help me with these simple question:

I am trying to forecast for 1 day into the future.

My dataset looks like this:

df:
Date Volume
Jan1 100
...
June1 99
June2 105
June3 90
June4 NaN

How do I forecast 1 day into the future (June4)?

I have tried the following as an example:

  test_data = ListDataset([{"start": df.index[0], 
                              "target": df.Volume[:"June4"]}], 
                            freq="D")

    estimator = NBEATSEstimator(freq="D", prediction_length=1, context_length = 5,trainer=Trainer(epochs=60,ctx="gpu"))
    predictor = estimator.train(training_data=test_data)

However, I get an error: 'Got NaN in first epoch. Try reducing initial learning rate.'_

**What should I do to forecast June4 if I have all previous data available (June3 and earlier)? What I am I doing wrong?**

Also, If I use target June3 instead (same dataset as above with data including June3 and June4 NaN value).

  test_data = ListDataset([{"start": df.index[0], 
                              "target": df.Volume[:"June3"]}], 
                            freq="D")

    estimator = NBEATSEstimator(freq="D", prediction_length=1, context_length = 5,trainer=Trainer(epochs=60,ctx="gpu"))
    predictor = estimator.train(training_data=test_data)

Forecasting results that I am getting are super close to June3 results.

**Does it simply replicates June3 results, or does it uses June2 and earlier and then tries to predict 1 day into the future (June3)?**

If I use this example:
https://github.com/awslabs/gluon-ts

It uses:
from gluonts.dataset.common import ListDataset
training_data = ListDataset(
    [{"start": df.index[0], "target": df.value[:"2015-04-05 00:00:00"]}],
    freq = "5min"
)
estimator = DeepAREstimator(freq="5min", prediction_length=12, trainer=Trainer(epochs=10))
predictor = estimator.train(training_data=training_data)

**What does it actually predicts? Does it predict 2015-04-14 23:55:00 - 2015-04-15 00:55:00 or 2015-04-15 00:00:00 - 2015-04-15 01:00:00? Charts seems to be 2015-04-14 23:55:00 - 2015-04-15 00:55:00**

Thank you!

**Comment by antonkiselevtdg** [comment]
What if you try to use June3 as target, but choose prediction_length=2?

**Comment by antonkiselevtdg** [comment]
What if you try to use June3 as target, but choose prediction_length=2?

**Comment by antonkiselevtdg** [comment]
What if you try to use June3 as target, but choose prediction_length=2?

**Comment by antonkiselevtdg** [comment]
What if you try to use June3 as target, but choose prediction_length=2?

**Comment by skybullet1987** [comment]
> What if you try to use June3 as target, but choose prediction_length=2?

Can definitely try that. However, I also wanted to better understand how "target' works and why it did not work.
